### ISSUE TITLE
Loxone Scenes disappeared after update to 0.25

### ISSUE THREAD

**Issue by knallt** [JoDehli/PyLoxone#34]
Hi, 

I really love this integration. Thank you for this.

I updated today to version 0.25 and after update all Loxone (light) scenes disappeared from my installation. Any idea why?

I have HA 0.114.4 installed, latest version.
Thx for any hint.

**Comment by JoDehli** [comment]
Do you see something on the log messages?

**Comment by knallt** [comment]
Hey, thx for quick reply. 

Hm, no nothing in the logs regarding loxone. 
The scenes just disappeared after update of HACS and the Loxone binding. So far can't find any errors and have no idea where they are...

**Comment by JoDehli** [comment]
@knallt Please try this log configuration and try it again. Maybe we some errors.

```ỳaml
logger:
  default: critical
  logs:
    homeassistant: debug
    custom_components.loxone: debug
    loxoneclient: debug
    loxoneclient.loxminiserver: debug
```

**Comment by JoDehli** [comment]
@knallt have you tried it?

**Comment by knallt** [comment]
Thx for asking again. Yes I did, but so far I could not manage to get more details. Scenes are still disappeared. I’ll try it again and post it here.

**Comment by knallt** [comment]
Hi Joachim, 
I don't get any further information. I also tried the logger debug configuration. I get warning but nothing is really wrong.
Also checked rights etc. 

e.g. warning:
`Logger: homeassistant.helpers.entity
Source: helpers/entity.py:357
First occurred: September 12, 2020, 9:35:57 AM (2 occurrences)
Last logged: 6:18:45 AM

Updating state for switch.steckdose_kuhlschrank (<class 'custom_components.loxone.switch.LoxoneTimedSwitch'>) took 8.106 seconds. Please report it to the custom component author.
Updating state for light.licht_spots_henri (<class 'custom_components.loxone.light.LoxoneDimmer'>) took 6.349 seconds. Please report it to the custom component author.`

Normally the plugin generate the scenes initially when starting up, right?

**Comment by JoDehli** [comment]
I think the problem is the loading time of your system. Can you increase the time from 0.2 to maybe 1 second or even more.

Change it in locally in your system for resting.
Here ist the link.to the line:

https://github.com/JoDehli/PyLoxone/blob/master/custom_components/loxone/scene.py#L46

**Comment by knallt** [comment]
Hey, 
tested it with 5 seconds. But scenes are still not loaded. Is there any option to push to reload scense (generate scences)?
In the configuration I have:

`loxone:
  port: 80
  host: 10.10.10.xy
  username: <user>
  password: <pw>
  generate_scenes: true`

Thank you!

**Comment by JoDehli** [comment]
You can try 2 more things:

1. disable all other integrations and try it again. do the scenes then come up?
2. download older version from here https://github.com/JoDehli/PyLoxone/tags and try. when you have found a working 
    version I can look what has changed. 

It is difficult to test because at my system I can not reproduce this error.

**Comment by knallt** [comment]
Thx. 
With Version 0.24 I can see the scenes again.

**Comment by JoDehli** [comment]
This is strange. I changed nothing on the scene in the new release. Maybe you can try the next release when I release it.

**Comment by knallt** [comment]
Hey Joachim,
Update: after Update to Home Assistant 0.115.1 all scenes from the Plugin/Loxone are back (also Version 0.25). Thanks again!
BTW Are you still planning to bring the Plugin to official HA without HACS?

**Comment by JoDehli** [comment]
@knallt I see no real benefit to make a official plugin. I see only the much more work for me. I started this for my own and shared it and try to solve as much problems I can. But the restrictions and rules to make it official are so big. And the support is not good from the core devs from homeassistant. It is properly because they have so much work to do. I understand that. But for me it is nearly impossible to do it myself. 

What I can do is to separate the communication part to a extra pip module. And maybe I can try it to get it in the official hacs repos. But in Homeassistant it is to much work. I tried it and they told me only what is wrong with the code but not how to solve it. 
Hope you understand that.
